ci/testkomodo.sh

```diff
@@ -15,6 +15,33 @@ install_test_dependencies () {
     pip install -r dev-requirements.txt
 }
 
+run_ert_with_opm () {
+    pushd "${CI_TEST_ROOT}"
+
+    mkdir ert_with_opm
+    pushd ert_with_opm || exit 1
+
+    cp "${CI_SOURCE_ROOT}/test-data/eclipse/SPE1.DATA" .
```

Comment:
just wondering, whether we should rename `/test-data/eclipse` into `/test-data/spe1` have two configs there; one for eclipse run and one for flow run?

Reply:
Voted down by tech lead.